frontend/src/app/feature/room/room-message/room-message.component.ts

```diff
@@ -19,14 +19,14 @@ export class RoomMessageComponent implements OnInit {
 
     getAuthorClass() {
         return {
-            system: this.message.author.isSystem(),
-            [this.message.author.color]: !this.message.author.isSystem()
+            system: this.message.author.system,
+            [this.message.author.color]: !this.message.author.system
```

Comment:
i know that i originally did this, but i think we can just hardcode red color into system player and delete this line and getMessageClass method

Reply:
Okay. But it should be a color from the schema. 'Warn' for example. red is ugly and will stay hardcoded for all schemas if we have a few of them in the future.

Reply:
Yup, i meant material red =)